dace/frontend/operations.py

```diff
@@ -105,7 +127,7 @@ def __call__(self, compiled_sdfg: 'CompiledSDFG', args: Tuple[Any, ...]):
         # Restore state after skipping contents
         compiled_sdfg.do_not_execute = old_dne
 
-        return ret
+        return None
```

Comment:
Why this change?

Reply:
> - Cleanup: remove unused variable for return value (didn't seem to be used)

The variable didn't seem serve any purpose. So basically: "while I'm already here, might as well..."